test.sh

```diff
@@ -12,6 +12,11 @@ go test -logtostderr=true
 t2=$?
 cd ..
 
+cd drivers
+go test -logtostderr=true
+t2=$?
```

Comment:
This reuses `t2` ; it shouldn't. If the previous assignment fails and this one succeeds, then CI will consider the tests to have passed when that is clearly not the case. Needs to be fixed.